### ISSUE TITLE
Irrelevant 

### ISSUE THREAD

**Issue by ghost** [LVPlayground/playground#602]

**Comment by Specifer** [comment]
Whoa , I just did it few days ago and it worked ?
How and where did you guys use it ? I did /inf and /lock, It worked with housma.

**Comment by TheRealSinned** [comment]
Whoa, well, lucky you. Hasn't worked for me once in the past few months. Both me and Halo tried the cmd. We used it on both an inf and an elegy. We tried it at 2 location. First one was right in front of the ship and then after he /fixed me we tried it near LVP Stadium. Not that the location matters anyway. If the cmd doesn't work at one place it won't work at another either.

**Comment by TheRealSinned** [comment]
Whoa, well, lucky you. Hasn't worked for me once in the past few months. Also your case is particularly interesting since the cmd can apparently only be used on "public vehicles". So using it on an /inf you spawned wouldn't really work. Unless this rule was added right after you tried it. Anyway both me and Halo tried the cmd. We used it on both an infernus and on an elegy(public ones, obviously). We tried it at 2 locations. First one was right in front of the ship and then after he /fixed me we tried it near LVP Stadium. Not that the location matters anyway. If the cmd doesn't work at one place it won't work at another either.
